packages/aws-cdk-lib/aws-ec2/test/instance-type.test.ts

```diff
@@ -0,0 +1,8 @@
+import { InstanceClass, InstanceSize, InstanceType } from '../lib';
```

Comment:
@moelasmar I've added simple unit test for `InstanceType` class. Is it enough?

Reply:
it looks good